app/src/main/java/org/mozilla/fenix/compose/home/HomeSectionHeader.kt

```diff
@@ -80,32 +86,37 @@ fun HomeSectionHeader(
 @Composable
 private fun HomeSectionHeaderContent(
     headerText: String,
-    description: String,
+    textColor: Color = FirefoxTheme.colors.textPrimary,
+    description: String = "",
     showAllTextColor: Color = FirefoxTheme.colors.textAccent,
     onShowAllClick: () -> Unit,
```

Comment:
Let's make `onShowAllClick` nullable and remove the need of `hasShowAllText`.

Reply:
I usually avoid using nullable lambdas, but I am good either way in this case.